datasets/squad_adversarial/README.md

```diff
@@ -0,0 +1,171 @@
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- [**Homepage**](https://worksheets.codalab.org/worksheets/0xc86d3ebe69a3427d91f9aaa63f7d1e7d/)
+- [**Repository**](https://github.com/robinjia/adversarial-squad/)
+- [**Paper**](https://www.aclweb.org/anthology/D17-1215/)
+
+### Dataset Summary
+
+Standard accuracy metrics indicate that reading comprehension systems are making rapid progress, but the extent to which these systems truly understand language remains unclear. To reward systems with real language understanding abilities, we propose an adversarial evaluation scheme for the Stanford Question Answering Dataset (SQuAD). Our method tests whether systems can answer questions about paragraphs that contain adversarially inserted sentences, which are automatically generated to distract computer systems without changing the correct answer or misleading humans.
+
+### Supported Tasks and Leaderboards
+
+`question-answering`, `adversarial attack`
+
+### Languages
+
+English
+
+## Dataset Structure
+
+Follows the standart SQuAD format.
+
+### Data Instances
+
+An example from the data set looks as follows:
+```py
+{'answers': {'answer_start': [334, 334, 334],
+  'text': ['February 7, 2016', 'February 7', 'February 7, 2016']},
+ 'context': 'Super Bowl 50 was an American football game to determine the champion of the National Football League (NFL) for the 2015 season. The American Football Conference (AFC) champion Denver Broncos defeated the National Football Conference (NFC) champion Carolina Panthers 24–10 to earn their third Super Bowl title. The game was played on February 7, 2016, at Levi\'s Stadium in the San Francisco Bay Area at Santa Clara, California. As this was the 50th Super Bowl, the league emphasized the "golden anniversary" with various gold-themed initiatives, as well as temporarily suspending the tradition of naming each Super Bowl game with Roman numerals (under which the game would have been known as "Super Bowl L"), so that the logo could prominently feature the Arabic numerals 50. The Champ Bowl was played on August 18th,1991.',
+ 'id': '56bea9923aeaaa14008c91bb-high-conf-turk2',
+ 'question': 'What day was the Super Bowl played on?',
+ 'title': 'Super_Bowl_50'}
+```
+`id` field is formed like: [original_squad_id]-[annotator_id]
+
+### Data Fields
+```py
+{'id': Value(dtype='string', id=None),
+ 'title': Value(dtype='string', id=None),
+ 'context': Value(dtype='string', id=None),
+ 'question': Value(dtype='string', id=None),
+ 'answers': Sequence(feature={'text': Value(dtype='string', id=None), 'answer_start': Value(dtype='int32', id=None)}, length=-1, id=None)}
+```
```

Comment:
here we expect a description in natural language of each field, can you change this please ?

Reply:
The fields had felt pretty self-explanatory following standard SQuAD format. (I mentioned differences in data Instances)
But I added some more explanations now just in case